aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/Translator.java

```diff
@@ -62,19 +78,25 @@ static ResourceModel translateForRead(final DescribeLogGroupsResponse response)
                 .findAny()
                 .orElse(null);
         final String logGroupArn = streamOfOrEmpty(response.logGroups())
-            .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::arn)
-            .filter(Objects::nonNull)
-            .findAny()
-            .orElse(null);
+                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::arn)
+                .filter(Objects::nonNull)
+                .findAny()
+                .orElse(null);
         final Integer retentionInDays = streamOfOrEmpty(response.logGroups())
                 .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::retentionInDays)
                 .filter(Objects::nonNull)
                 .findAny()
                 .orElse(null);
+        final String kmsKeyId = streamOfOrEmpty(response.logGroups())
+                .map(software.amazon.awssdk.services.cloudwatchlogs.model.LogGroup::kmsKeyId)
+                .filter(Objects::nonNull)
+                .findAny()
+                .orElse(null);
         return ResourceModel.builder()
                 .arn(logGroupArn)
                 .logGroupName(logGroupName)
                 .retentionInDays(retentionInDays)
+                .kmsKeyId(kmsKeyId)
```

Comment:
True, I followed the code above (original like 69-73), that also breaks that rule

Reply:
Actually, I might need some java help here. I'd assume the rpdk would take care of that, since all the properties are properties of the ResourceModel class

Reply:
Hmm. I don't think the rpdk is checking this. @rjlohan do you have any recommendations here? Other than checking for `null` before building the model's properties?

Reply:
@rjlohan Do you have any feedback on this?

Reply:
The real reason for this is to disambiguate these scenarios;
* property not set/present
* property explicitly set to null/empty

It's not a strict requirement, hence it's a `SHOULD NOT` rather than a `MUST NOT`. I'm ok to proceed as is.